server/base/src/main/java/org/apache/accumulo/server/fs/VolumeManagerImpl.java

```diff
@@ -351,15 +347,6 @@ public void bulkRename(Map<Path,Path> oldToNewPathMap, int poolSize, String pool
       }
       return null;
     })));
-    workerPool.shutdown();
-    try {
-      while (!workerPool.awaitTermination(1000L, TimeUnit.MILLISECONDS)) {}
-      for (Future<Void> future : results) {
-        future.get();
-      }
```

Comment:
Want to keep this code that waits on the futures.

Reply:
Are you sure it is necessary to block `bulkRename` method?